Makefile

```diff
@@ -86,6 +87,15 @@ V ?= $(or $(VERBOSE),0)
 Q = $(if $(filter 1,$V),,@)
 H = $(shell printf "\033[34;1m=>\033[0m")
 
+# To build Pilot, Mixer and CA with debugger information, use DEBUG=1 when invoking make
+DEBUG ?= 0
+ifeq (${DEBUG},1)
+	export GCFLAGS = -N -l
+    $(info $(H) Build with debugger information)
+else ifneq ($(DEBUG),0)
+    $(error DEBUG has a default value of 0 for regular build and 1 for debugging)
```

Comment:
I don't care strongly on this point, but the msg isn't as explicit about the issue.  I'm thinking more like:

$(error "DEBUG must either be 0 for release build (default) or 1 for debug build")